local.tf

```diff
@@ -112,8 +112,8 @@ locals {
     cpu_credits                          = "standard"                                                          # T2/T3 unlimited mode, can be 'standard' or 'unlimited'. Used 'standard' mode as default to avoid paying higher costs
     market_type                          = null
     metadata_http_endpoint               = "enabled"  # The state of the metadata service: enabled, disabled.
-    metadata_http_tokens                 = "optional" # If session tokens are required: optional, required.
-    metadata_http_put_response_hop_limit = null       # The desired HTTP PUT response hop limit for instance metadata requests.
+    metadata_http_tokens                 = "required" # If session tokens are required: optional, required.
+    metadata_http_put_response_hop_limit = 1          # The desired HTTP PUT response hop limit for instance metadata requests.
```

Comment:
These changes will trigger an update for existing users unless they provide these values already.

I propose we group such changes for the next major release and make it visible in the upgrade guide.

WDYT?

Reply:
Yes, sure, fully agree.